### PR TITLE
Duplicate entries check enforced

### PR DESCRIPTION
**Pull request by sebbalex**
This aims to enforce the duplicate entries check, it will prevents at any registration step to insert data already present in database.
This will also mitigate #83

### CODE REVIEW

File: src/tpl/register-confirm.html [hunk 1]
```diff
@@ -3,8 +3,10 @@
         <div class="container">
             <div class="row">
                 <div class="my-5 col-md-8">
-
+                    
+                    {{^errorMsg}}
                     <h3>Confermare registrazione dell'URL nel database?</h3>
+                    {{/errorMsg}}
```
Comment (bfabio) [inline src/tpl/register-confirm.html:9]: Maybe we should show the error if the entry is duplicated.

Comment (libremente) [reply src/tpl/register-confirm.html:9]: +1 on this, it can be a nice hint. Also, this is public info so we got nothing to hide :smiley:

Comment (sebbalex) [reply src/tpl/register-confirm.html:9]: error is already there, this template inherit the default layout where the error section is placed: [see here](https://github.com/italia/developers-italia-onboarding/blob/master/public/index.html#L117)

Comment (bfabio) [reply src/tpl/register-confirm.html:9]: @sebbalex sorry, I missed that :+1:

**Review by bfabio — APPROVED** [review]
:+1:

**Pull request merged**